docs/advanced_topics/settings.rst

```diff
@@ -317,6 +317,18 @@ Case-Insensitive Tags
 
 Tags are case-sensitive by default ('music' and 'Music' are treated as distinct tags). In many cases the reverse behaviour is preferable.
 
+Multi-word tags
+---------------
+
+.. code-block:: python
+
+  TAGGIT_SPACES_ALLOWED = False
+
+Tags can only consist of a single word, no spaces allowed. The default behaviour is the reverse
```

Comment:
I found the phrase "The default behaviour is the reverse" slightly confusing. How about "The default setting is `True` (spaces in tags are allowed)"?